pydoc/developer/BENCHMARK.md

```diff
@@ -0,0 +1,470 @@
+# PyTest Benchmarks
+
+**Purpose**: Outline the general steps and options available for running the new PyTest benchmarks
+
+The PyTest benchmark structure allows the flexibility of running the entire suite, a single file, 
+or a single benchmark.
+
+## Running The Full Suite
+In most cases, running the full benchmark suite is desired. The simplest way to do this is to navigate into the 
+root-level of arkouda and run `make benchmark`
+
+This will run the entire benchmark suite with the following command:
+```commandline
+python3 -m pytest -c benchmark.ini --benchmark-autosave --benchmark-storage=file://benchmark_v2/.benchmarks
+```
+
+## Running Single Files or Tests
+
+In instances where a single test or set of tests needs to be ran, use the `-k <name_of_file>.py` argument. For these 
+examples, `encoding_benchmark.py` and the specific benchmark from within the file `bench_encode` is used.

```

Comment:
```suggestion
In instances where a single test or set of tests needs to be ran, use the `-k <name_of_file>.py` argument. 
```

Following the above, I would add the information the first example will run all the benchmarks in the `encoding_benchmark.py` file.